modules/ROOT/pages/configuring-dev-env.adoc

```diff
@@ -53,3 +54,191 @@ include::{partialsdir}/attributes.adoc[]
 == Adding Adding AeroGear SDK modules
 
 
+[role="primary"]
+.Android
+
+****
+
+*Prerequisites*
+
+You will need the following installed on your system
+
+ * Android SDK or Android Studio from https://developer.android.com/studio/index.html[Google]
+
+It is recommended you have the following installed
+
+ * The latest version of http://openjdk.java.net/install/index.html[OpenJDK]
+
+*Using a Published SDK*
+
+_jcenter()_
+
+To use a published version of the SDK you have to add jcenter to the build.grade file in your project's root directory
+
+[source, groovy]
+----
+allprojects {
+    repositories {
+        jcenter() // <-- Add This line
+        google()
+    }
+}
+----
+
+*Reference the Dependencies*
+
+In the build.gradle file of your app you can begin adding dependencies provided by the SDK.
+
+[source,groovy]
+----
+dependencies {
+    ...
+    implementation 'org.aerogear:android-push:[version]'
+    implementation 'org.aerogear:android-auth:[version]'
+    implementation 'org.aerogear:android-security:[version]'
+}
+----
+
+*Additional Dependencies*
+
+For push you also need to add a dependency to the Firebase SDK:
+
+[source,groovy]
+----
+dependencies {
+    ...
+    implementation "com.google.firebase:firebase-messaging:[version]"
+}
+----
+
+A full list of releases can be viewed https://github.com/aerogear/aerogear-android-sdk/releases[here]
+
+*Certificate Pinning*
+
+To implement certificate pinning in the individual SDKs, see link:certificate-pinning.html[the certificate pinning guide].
+
+****
+
+[role="secondary"]
+.iOS
+
+****
+
+*Prerequisites*
+
+To use the AeroGear SDK for iOS, you will need the following installed on your development machine:
+
+* iOS 9 or later
+* Xcode 9 or later
+* link:https://cocoapods.org/[CocoaPods]
+
+To get started with the AeroGear iOS SDK, you can set up the SDK in your existing or new project. The SDK is written in Swift.
+
+*CocoaPods*
+
+. The AeroGear SDK for iOS is available using the CocoaPods package manager.
+If you have not installed CocoaPods, install it by running the command:
++
+[source,bash]
+----
+gem install cocoapods
```

Comment:
Implies having decent version of ruby. Standard one is kinda too old.
Best to mention ruby as dependency.